jitbuilder/release/src/Simple.cpp

```diff
@@ -89,7 +89,8 @@ SimpleMethod::buildIL()
    cout << "SimpleMethod::buildIL() running!\n";
    Return(
       Add(
-         Load("value"),
+         Copy(
```

Comment:
Sorry, Charlie, I just realized that we probably shouldn't be mucking with Simple.cpp because the jitbuilder blog articles (and several of the presentations) reference it and describes it in detail. So I'd prefer not to just add Copy here (though it didn't register with me in my first review).

Given you have OperandArrayTests which presumably exercise Copy, I'm not sure we *really* need it in the code samples.